### PR TITLE
Deveplement of POST /enrich request

### PR DESCRIPTION
**Pull request by LMFonseca**
Processing and reply with the right contact (without filtering)

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -17,6 +20,14 @@
...
-get '/enrich' do
-  Services::ContactsEnricher.new.enrich('contact_id_1')
+
+post '/enrich' do
+  
+  contact_id = @request_payload["contact"]["id"]
+  fields = @request_payload["contact"]["fields"]
+
+  puts contact_id
+  puts fields
```
Comment (bmpj13) [inline app.rb:30]: Can we remove this `puts`? 👍

File: services/contacts_enricher.rb [hunk 2]
```diff
@@ -2,9 +2,13 @@
...
 require_relative '../repositories/contacts.rb'
 
+require 'json'
+
+
 module Services
   class ContactsEnricher
-    def initialize(options: {})
+    def initialize(options = {})
+      puts options.fetch(:fields, [])
```
Comment (bmpj13) [inline services/contacts_enricher.rb:11]: Same as above

File: app.rb (hunk 1 above)
Comment (LMFonseca) [reply app.rb]: It's done!

**Review by bmpj13 — APPROVED** [review]
LGTM 🏅

**Review by DanielCPina — APPROVED** [review]
Looks good to me

**Pull request merged**